baybe/parameters/numerical.py

```diff
@@ -150,5 +150,38 @@ def summary(self) -> dict:
         return param_dict
 
 
+@define(frozen=True, slots=False)
+class _FixedNumericalContinuousParameter(ContinuousParameter):
+    """Parameter class for fixed numerical parameters."""
+
+    is_numeric: ClassVar[bool] = True
+    # See base class.
+
+    value: float = field(converter=float)
+    """The fixed value of the parameter."""
+
+    @property
+    def bounds(self) -> Interval:
+        """The value of the parameter as a degenerate interval."""
+        return Interval(self.value, self.value)
+
+    @override
+    def is_in_range(self, item: float) -> bool:
+        return item == self.value
+
+    @override
+    @property
+    def comp_rep_columns(self) -> tuple[str, ...]:
```

Comment:
no, difference between mutable and immutable containers. Here, we have potentially multiple columns. In case of a list, it would be `list[str]` but `tuple[str]` means tuple with *exactly* on string element

Reply:
`exactly one string element` is exactly what this function appears to return, hence my suggestion

Reply:
Ah, now I see what you mean. Hm, this is an override of the base class method, which has multiple elements. But I guess specializing it to a subset type should be doable / Liskov 🤔 @Waschenbacher: you can give it a try and see if mypy complains